cardano-cli/src/Cardano/CLI/Run.hs

```diff
@@ -48,16 +49,20 @@ data ClientCommand =
     -- now-deprecated \"shelley\" subcommand.
   | DeprecatedShelleySubcommand ShelleyCommand
 
+  | CliPingCommand PingCmd
+
   | forall a. Help ParserPrefs (ParserInfo a)
   | DisplayVersion
 
 data ClientCommandErrors
   = ByronClientError ByronClientCmdError
   | ShelleyClientError ShelleyCommand ShelleyClientCmdError
```

Comment:
This should be under `ShelleyCommand` because we can't bring Byron nodes with this command.

Reply:
I tried running this against a Byron BFT node set up with `scripts/byron-to-alonzo/mkfiles.sh byron` and I got this result:

```
$ CARDANO_NODE_SOCKET_PATH=example/main.sock cardano-cli query tip --testnet-magic 42
{
    "block": 263,
    "epoch": 2,
    "era": "Byron",
    "hash": "3a64c8302f4c46b9ed80b1a91e950d09d2918e4d6554ff05ac6377312acdc8a7",
    "slot": 262,
    "syncProgress": "100.00"
}
$ cardano-cli ping -h localhost -p 3001 -q -j -c 2 -m 42
{ "pongs": [ {"cookie":0,"host":"127.0.0.1:3001","max":1.64e-4,"mean":1.64e-4,"median":1.64e-4,"min":1.64e-4,"p90":1.64e-4,"sample":1.64e-4,"timestamp":"2022-12-10T04:01:04.334246Z"},
{"cookie":1,"host":"127.0.0.1:3001","max":4.81e-4,"mean":3.225e-4,"median":3.225e-4,"min":1.64e-4,"p90":4.493e-4,"sample":4.81e-4,"timestamp":"2022-12-10T04:01:05.335669Z"}] }
```

Reply:
This tool is not era depended, so neither Byron nor Shelley looks good to me.  What about adding it to `Miscellaneous commands`, or maybe it's own category of `Network commands`?